server/sonar-db-migration/src/main/java/org/sonar/server/platform/db/migration/version/v70/DbVersion70.java

```diff
@@ -31,7 +31,7 @@ public void addSteps(MigrationStepRegistry registry) {
       .add(1901, "Populate QUALITY_GATES.IS_BUILT_IN", PopulateQualityGatesIsBuiltIn.class)
       .add(1902, "Make QUALITY_GATES.IS_BUILT_IN not null", MakeQualityGatesIsBuiltInNotNullable.class)
       .add(1903, "Remove quality gates loaded templates", RemoveQualityGateLoadedTemplates.class)
-    ;
+      .add(1904, "Drop LOADED_TEMPLATES table", DropLoadedTemplatesTable.class);
```

Comment:
Minor : I loved the fact that the semicolon is on a separate line for solving merge conflicts.

Reply:
Yes, me too, but the reformat method from Intellij is re-setting the semicolon at the end of previous line